nautobot/core/templates/inc/paginator.html

```diff
@@ -1,34 +1,42 @@
 {% load helpers %}
 
-<div class="paginator text-right">
+<div class="paginator text-end">
     {% if paginator.num_pages > 1 %}
         <nav>
-            <ul class="pagination pull-right">
+            <ul class="pagination float-end">
                 {% if page.has_previous %}
-                    <li><a href="{% querystring request page=page.previous_page_number %}"><i class="mdi mdi-chevron-double-left"></i></a></li>
+                    <li class="page-item">
+                        <a class="page-link" href="{% querystring request page=page.previous_page_number %}" aria-label="Previous">
+                          <span aria-hidden="true">&laquo;</span>
```

Comment:
Is there a reason we went from the MDI chevrons to the literal `&laquo;`/`&raquo;`? Part of the design?